### PR TITLE
Update examples to use LOOKER_CORE_STANDARD_ANNUAL edition

### PR DESCRIPTION
**Pull request by efeelaiho**
Currently we are blocking/not provisioning LOOKER_CORE_STANDARD ( Pay as you go) with no timeline as to when it'll be back. These should resolve failed tests for: 

Fixes https://github.com/hashicorp/terraform-provider-google/issues/16715
Fixes https://github.com/hashicorp/terraform-provider-google/issues/17592

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none
```

### CODE REVIEW

File: mmv1/templates/terraform/examples/looker_instance_full.tf.erb [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
   admin_settings {
     allowed_email_domains = ["google.com"]
   }
   // User metadata config is only available when platform edition is LOOKER_CORE_STANDARD.
-  user_metadata {
-    additional_developer_user_count = 10 
-    additional_standard_user_count  = 10
-    additional_viewer_user_count    = 10
-  }
+  // user_metadata {
```
Comment (c2thorn) [inline mmv1/templates/terraform/examples/looker_instance_full.tf.erb:10]: should this just be deleted?

Comment (efeelaiho) [reply mmv1/templates/terraform/examples/looker_instance_full.tf.erb:10]: Sure! I'll just add it back if needed

File: .ci/infra/terraform/README.md [hunk 2]
```diff
@@ -57,6 +57,7 @@ Quotas that will need to be adjusted to support all tests:
 - Cloud Workstation cluster quota in `us-central1`
 - VMWare Engine nodes per region in `southamerica-west1`
 - VMWare Engine nodes across regions
+- Looker `StandardSubscriptionInstancesPerProjectPerRegion`. This must be requested manually from their team, and can't be self-served in the Cloud Console.
```
Comment (c2thorn) [inline .ci/infra/terraform/README.md:60]: Our CI naively stops all builds that edit `.ci/` files. Do you mind separating this out into a separate PR so we can run the test changes?

Comment (efeelaiho) [reply .ci/infra/terraform/README.md:60]: Sure!

**Comment by efeelaiho** [comment]
@c2thorn Can we runs the tests again? I've added quota to the projects to resolve:

```
{
  "error": {
    "code": 429,
    "message": "Quota limit 'StandardSubscriptionInstancesPerProjectPerRegion' has been exceeded. Limit: 0 in region us-central1.",
    "status": "RESOURCE_EXHAUSTED",
    "details": [
      {
        "@type": "type.googleapis.com/google.rpc.QuotaFailure",
        "violations": [
          {
            "subject": "project:1067888929963",
            "description": "Quota 'StandardSubscriptionInstancesPerProjectPerRegion' exhausted. Limit 0 in region us-central1"
          }
        ]
      }
    ]
  }
}
```

**Review by c2thorn — APPROVED** [review]

**Pull request merged**